src/source/DataModelV2/SoundService.cpp

```diff
@@ -67,6 +67,30 @@ SoundService::SoundService()
 	stockSound->setName("Page");
 	stockSound->setSoundId("/content/sounds/pageturn.wav");
 	stockSound->setParent(this);
+
+	// Click
+	stockSound = new SoundInstance();
```

Comment:
I think these should be deleted in the destructor, as I don't see them being deleted anywhere else

Reply:
Not sure what you mean by that, could you elaborate? Sorry if this is a noobish question